[ST-348] Add initial sequence recorder docs (#474)
cobalthq/cobalt-product-public-docs: content/en/Platform Deep Dive/Scans/sequence-recorder.md

> # Sequence recorder

Reviewer: ```suggestion
# Sequence Recorder
```

Cobalt's Sequence Recorder is a browser plugin that records your actions in the browser. This can specifically
help you with setting up an authenticated target. These actions will then be

> replayed by the scans when trying to authenticate to the target.

Reviewer: ```suggestion
replayed by the DAST Scanner when trying to authenticate to the target.
```

# Installation

> _Work in progress_

Reviewer: So we shouldn't merge yet?

Author: I want to merge it, these steps well come once we have the app published. It's a bit of a chicken and egg problem 🐣

# Usage

> Once installed, click on Cobalt's extension icon, a small window will pop up. Type in there

Reviewer: ```suggestion
Once installed, click on Cobalt's extension icon, and a small window will pop up. Type in there
```

> the URL of the target that you'd like to add authentication to and click `Start Recording`

Reviewer: ```suggestion
the URL of the target that you'd like to add authentication to and click `Start Recording`.
```

![Click Start recording](/deepdive/scans/sequence-recorder/recorder1.png "Start Recording")

> You'll see a red dot (🔴) in the tab title, meaning that it's recording. Perform the necessary

Reviewer: I believe it is a circle. At least it is the name of this icon.
```suggestion
You'll see a red circle (🔴) in the tab title, meaning that it's recording. Perform the necessary
```

actions to authenticate. Once done, click on the extension icon again and click `Stop Recording`.

![Click Stop recording](/deepdive/scans/sequence-recorder/recorder2.png "Stop Recording")

A new tab will appear. You can either copy the sequence to the clipboard or download it as a file.

![Get the sequence](/deepdive/scans/sequence-recorder/recorder3.png "Get the sequence")

> You can now go to your target settings on Cobalt and paste or upload the sequence there.

Reviewer: App? Platform? 🤔 

```suggestion
You can now go to your target settings on Cobalt's platform and paste or upload the sequence there.
```

> Once those steps are completed, scans against this target will try to authenticate using these steps.

Reviewer: Not 100% sure about this one. I thought why use `those` and `these` for the same steps.
```suggestion
Once these steps are completed, scans against this target will try to authenticate using these steps.
```
